src/main/webapp/index.html

```diff
@@ -8,7 +8,47 @@
 
 <div id="navbar-container"></div>
 <div class="container">
-  <h1 class="display-2 mb-5 mt-5">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
+  <h1 class="display-2 mb-5 mt-5 text-center">The Open Source Community is for <span class="emphasis">Everyone</span></h1>
+  <!-- TODO: Change button text/link depending on user authorization. -->
+  <div class="d-flex justify-content-center"><div class="btn btn-emphasis mt-5 mb-5 text-center btn-lg btn-block btn-main">Create a Profile!</div></div>
+</div>
+<div class="emphasis-box mt-5 mb-5">
+  <div class="container">
+    <h3 class="display-3 emphasis mt-3 mission strong">About Us: </h3>
+    <p class="mission lead mb-3">
+      OpenSesame is a mentorship platform, connecting interested contributors with experienced developers to help newcomers start working on projects. 
+      Whether you are looking to explore new ventures, get new contibutors onboard, or get advice from someone more experienced, OpenSesame can help you make connections within the open source community.
+    </p>
+  </div>
+</div>
+<div class="container">
+  <div class="row mt-5">
+    <div class="col-md-4 border bg-light">
+      <h2 class="emphasis mt-4 mb-4 strong">Looking to explore?<h2>
+    </div>
+    <div class="col-md-4 border bg-light">
+      <h2 class="emphasis mt-4 mb-4 strong">Ready to connect?<h2>
+    </div>
+    <div class="col-md-4 border bg-light">
+      <h2 class="emphasis mt-4 mb-4 strong">Want to be a mentor?<h2>
+    </div>
+    <div class="w-100"></div>
+    <div class="col-md-4 border">
+      <p class="lead mt-3">
+        Check out the <a href="/mentors.html">Mentors</a> and <a href="/projects.html">Projects</a> pages to browse available opportunities!
+      </p>
+    </div>
+    <div class="col-md-4 border">
+      <p class="lead mt-3">
+        Once you create a profile and sign in, you can send an introduction email to any mentor on the <a href="/mentors.html">Mentors</a> or <a href="/projects.html">Projects</a> pages.
+      </p>
+    </div>
+    <div class="col-md-4 border">
+      <p class="lead mt-3">
+        Create a profile, sign in, and then you will be able to visit the Dashboard page. On the dashboard, click the 'Become a Mentor' button and input a project you work on to be added to our mentor database.
+      </p>
+    </div>
```

Comment:
This doesn't collapse properly on mobile. Consider using a single row of [cards with headers](https://getbootstrap.com/docs/4.0/components/card/#header-and-footer) here instead of two rows. Let's get this fixed before we merge:

[Screenshot of scaling issue](https://screenshot.googleplex.com/3iY0JvyE0Bc)

Reply:
Happy this is resolved, thanks!